src/main/java/org/isf/examination/service/ExaminationIoOperationRepository.java

```diff
@@ -38,5 +38,8 @@ public interface ExaminationIoOperationRepository extends JpaRepository<PatientE
 	List<PatientExamination> findByPatient_CodeOrderByPexDateDesc(@Param("patientCode") int patientCode);
 
 	@Query(value = "select p from PatientExamination p where p.patient.code = :patientCode order by p.pex_date desc")
-	Page<PatientExamination> findByPatient_CodeOrderByPexDateDesc(@Param("patientCode") int patientCode, Pageable pageable);
+	List<PatientExamination> findByPatient_CodeOrderByPexDateDesc(@Param("patientCode") int patientCode, Pageable pageable);
+	
+	@Query(value = "select p from PatientExamination p where p.patient.code = :patientCode order by p.pex_date desc")
+	Page<PatientExamination> findByPatient_CodeOrderByPexDateDescs(@Param("patientCode") int patientCode, Pageable pageable);
```

Comment:
I don't understand this change, is not exactly the same method only adding one "s" at the end?

Reply:
Sorry, my bad, I misreaded the returning type.

Reply:
BTW is the "s" really needed?

Reply:
okay